### PR TITLE
Describe how to Create pull requests

### PR DESCRIPTION
**Pull request by vipermessy**
#### Hello 404Xavier

I added the details, the process before making a Pull Request, hope you find this useful,

**Thanks in Advance**
_Viper Messy._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,4 +14,17 @@
...
+
+       git push -u origin  <feature_branch>
+      ```
+    2. After committing the feature branch, a new pull request appears. You can now compare the differences between the the two branches, in this case the PULL-REQUESTS branch and the main branch of the repository you forked from. It uses the git diff command to compare the two branches. It shows  files have been added, removed, deleted, changed and so on.
+      ```git
+
+         git diff main PUll-REQUESTS 
+         
+      ````
+
```
Comment (ndejoma) [inline README.md:37]: I would like you describe the process better, then I merge the chnages

**Comment by vipermessy** [comment]
I would like your suggestion, on how to improve.

**Review by ndejoma — CHANGES_REQUESTED** [review]
I would like you describe the process better.

**Review by vipermessy — COMMENTED** [review]
I am working on the changes right now. I will inform you when I am done.

File: README.md (hunk 1 above)
Comment (vipermessy) [reply README.md:37]: working on it

Comment (vipermessy) [reply README.md:37]: Made the appropriate changes, could you check it out.

**Review by ndejoma — APPROVED** [review]
Well done Viper, changes accepted

**Comment by ndejoma** [comment]
Changes accepted.

**Comment by ndejoma** [comment]
I am now merging the changes to the main branch

**Comment by ndejoma** [comment]
Changes merged, thank you for your contribution

**Pull request merged**